packages/foodie-backend/src/loaders/express.ts

```diff
@@ -0,0 +1,38 @@
+import * as express from 'express';
+import compression from 'compression';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+import express, { Request, Response } from 'express';
+import helmet from 'helmet';
+import AuthRouter from './app/auth/AuthRouter';
+import UserRouter from './app/user/UserRouter';
+// import VendorRouter from './app/vendor/VendorRouter';
+
+
+export default ({ app }: { app: express.Application }) => {
```

Comment:
why this way ?
I think the previous code I much easier to understand

Reply:
Ok, we revert to the one we can both understand. Is just a way to let typescript know that it is an express app